hook-consumer/src/main.rs

```diff
@@ -23,15 +23,28 @@ async fn main() -> Result<(), WebhookConsumerError> {
     .await
     .expect("failed to initialize queue");
 
-    let consumer = WebhookConsumer::new(
-        &config.consumer_name,
-        &queue,
-        config.poll_interval.0,
-        config.request_timeout.0,
-        config.max_concurrent_jobs,
-    )?;
-
-    let _ = consumer.run().await;
+    match config.transactional {
+        true => {
```

Comment:
I dunno why, but `match` on a `bool` cracks me up. We're living in the future. :)

Reply:
Aaah, I may be rusting too hard.